pom.xml

```diff
@@ -25,14 +25,14 @@
 
   <developers>
     <developer>
-      <name>Michael Wurster</name>
-      <email>miwurster@googlemail.com</email>
+      <name>Ben Wen</name>
```

Comment:
You should add yourself to the list of developers, not overwrite existing entries ;-)

Reply:
ok，thanks for your reminder.